docs/en/README.md

```diff
@@ -0,0 +1,83 @@
+# Getting started
+
+## Requirements
+
+- The minimum version of PHP required by this package is `8.0`.
+- `PDO` PHP extension.
+
+## Installation
+
+The package could be installed with composer:
+
+```
+composer require yiisoft/log-target-db --prefer-dist
+```
+
+## Migration
+
+The package provides a migration that creates the cache table for default `{{%log}}`. You can use it as follows:
```

Comment:
Need add files with raw sql for direct create tables and add information about it in this readme